src/main/java/Duke.java

```diff
@@ -19,22 +19,45 @@ public static void main(String[] args) {
         Task[] list = new Task[100];
         int index = 1;
         String input = scan.nextLine();
+        
         while (!input.equals("bye")) {
-            //Adding the command "list" to list all added tasks
-            if (input.equals("list")) {
-                for (int i = 1; i < index; i++) {
-                    System.out.println(list[i]);
-                }
-            } else if (input.contains("done")){
-                String arr[] = input.split(" ");
-                int num = Integer.parseInt(arr[1]);
-                System.out.println(list[num].done());
-            } else {
-                //Adds input to list
-                list[index] = new Task(input, index);
-                index++;
-                System.out.println("added: " + input);
+            String arr[] = input.split(" ",2 );
+            String command = arr[0];
+            switch (command) {
+                case "list":
+                    System.out.println("Here are the tasks in your list:");
+                    for (int i = 1; i < index; i++) {
+                        System.out.println(i + "." + list[i]);
+                    }
+                    break;
+                case "done":
+                    int num = Integer.parseInt(arr[1]);
+                    System.out.println(list[num].done());
+                    break;
+                case "todo":
+                    list[index] = new Task(arr[1]);
+                    System.out.println("Got it. I've added this task:\n" + list[index]);
+                    System.out.println("Now you have " + index + " tasks in the list");
+                    index++;
+                    break;
+                case "deadline":
+                    String[] deadline = arr[1].split("/by ", 2);
+                    list[index] = new Deadline(deadline[0], deadline[1]);
+                    System.out.println("Got it. I've added this task:\n" + list[index]);
+                    System.out.println("Now you have " + index + " tasks in the list");
+                    index++;
+                    break;
+                case "event":
+                    String[] event = arr[1].split("/at ", 2);
+                    list[index] = new Event(event[0], event[1]);
+                    System.out.println("Got it. I've added this task:\n" + list[index]);
+                    System.out.println("Now you have " + index + " tasks in the list");
+                    index++;
+                    break;
+                default:
+                    System.out.println("Invalid command");
```

Comment:
Good job use switch-case statements instead of if-else loops here to make the different scenarios clearer to the reader!